src/utils/HelperMethods.js

```diff
@@ -0,0 +1,90 @@
+import sgMail from '@sendgrid/mail';
+// import Mailgen from 'mailgen';
+// import {config} from 'dotenv';
+// const sgMail = require('@sendgrid/mail');
+require('dotenv').config();
```

Comment:
Line 5 is redundant. `dotenv.config()` is called once and from the entry file. That way, we wouldn't have to litter our code with many `dotenv.config()`s.

If you need more clarification check the dotenv [docs](https://www.npmjs.com/package/dotenv#usage) and maybe this [conversation](https://github.com/andela/rogue-backend/pull/23#discussion_r316191703)

Please ensure that `dotenv.config()` is called once and from the entry file (src/index.js).

Reply:
Hi Chidozie, thank you for your feedback.